types/field_type.go

```diff
@@ -1448,3 +1449,17 @@ func ConvertBetweenCharAndVarchar(oldCol, newCol byte) bool {
 	return (IsTypeVarchar(oldCol) && newCol == mysql.TypeString) ||
 		(oldCol == mysql.TypeString && IsTypeVarchar(newCol) && collate.NewCollationEnabled())
 }
+
+// IsTooBigFieldLength check if the varchar type column exceeds the maximum length limit.
+func IsTooBigFieldLength(colDefTpFlen int, colDefName, setCharset string) error {
```

Comment:
I suggest having `varchar` in the function name, since it is used exclusively for varchar.

Reply:
Updated.